src/components/layout/dashboard/DashboardPanels.js

```diff
@@ -54,9 +53,9 @@ const DashboardPanels = () => {
             </TabList>
             <TabPanels>
                 <TabPanel>
-                    <div className='dashboard__panels'>
+                    <SimpleGrid gap="3" columns={{base: "1", md:"2", xl:"3"}}>
```

Comment:
I have approved and merged the PR, but I'll leave some pedantic comments here :)
1. Configure your editor/IDE to run a formatter on save. The spaces between keys and values are not consistent.
2. When the values are numbers, better to use them unquoted

ie. 
```suggestion
                    <SimpleGrid gap="3" columns={{base: 1, md: 2, xl: 3}}>
```